types/lib/query-interface.d.ts

```diff
@@ -473,7 +473,7 @@ export class QueryInterface {
     records: object[],
     options?: QueryOptions,
     attributes?: string[] | string
-  ): Promise<object>;
+  ): Promise<object | number>;
```

Comment:
When does it return an object and when does it return a number? And by the way what is this object/number? I couldn't find in the docs :sweat_smile: